src/components/SiteMenuSider.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div
+    class="fixed top-0 left-0 z-50 w-full h-screen bg-black bg-opacity-[30%]"
+  >
+    <div
+      class="flex flex-col items-center gap-[22px] w-full bg-white px-[18px] py-[42px]"
+    >
+      <div
+        class="flex items-center justify-between w-full text-like-green mb-[40px]"
+      >
+        <NuxtLink
+          class="w-[90px] transition-transform hover:scale-105 active:scale-100"
+          :disabled="localeLocation(getHomeRoute).name === $route.name"
+          :to="localeLocation(getHomeRoute)"
+        >
+          <Logo class="fill-current" />
+        </NuxtLink>
+        <div
+          class="flex justify-center items-center cursor-pointer w-[40px] h-[40px]"
+          @click="close"
+        >
+          <IconClose class="w-[20px]" />
+        </div>
```

Comment:
Can we use ButtonV2 with plain preset and small size?

Reply:
[🎨 Update close button using ButtonV2](https://github.com/likecoin/liker-land/pull/1724/commits/f0b68d6d8823301501f42326e75109a29ea9352d)